package.json

```diff
@@ -187,8 +186,9 @@
     "start:mock": "MOCK=true npm run start:proxy",
```

Comment:
I think we need to copy the part for the mock routes[0] over as well, to not break this here. I'm not sure if this is still in use by someone, but let's be safe. I thin the cypress tests may use this.

[0] https://github.com/RedHatInsights/insights-inventory-frontend/blob/master/config/dev.webpack.config.js#L26-L39